### PR TITLE
Update doc-validator

### PR DESCRIPTION
**Pull request by jdbaldry**
https://github.com/grafana/technical-documentation/blob/main/tools/cmd/doc-validator/CHANGELOG.md contains the full list of changes.

You can see the failures in https://github.com/grafana/mimir/actions/runs/4895741370/jobs/8741701493?pr=4943.

### CODE REVIEW

File: docs/docs.mk
Comment (osg-grafana) [inline docs/docs.mk:104]: Nice catch!

File: docs/make-docs
Comment (osg-grafana) [inline docs/make-docs:2]: Love this self-documentation. :)

**Review by osg-grafana — APPROVED** [review]
Unblocking with the assumption that another (engineering) maintainer will have a look and over another approval.

**Comment by pracucci** [comment]
Hi @jdbaldry. Out of curiosity, what's the state of this PR?

**Comment by jdbaldry** [comment]
Closing in favor of #5172

**Pull request closed without merging**
